specification/billing/resource-manager/Microsoft.Billing/preview/2020-11-01-preview/billingPromotions.json

```diff
@@ -172,6 +172,42 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Billing/promotions": {
+      "get": {
+        "tags": [
+          "Promotions_CheckEligibility"
+        ],
+        "operationId": "Promotions_CheckEligibility",
```

Comment:
>Promotions_CheckEligibility [](http://example.com/codeflow?start=24&length=27)

The operation id should be "Promotions_List".

Reply:
This operation is not for listing promotions resources, but instead it check eligibility of the promotions within a given subscription